### PR TITLE
added multiplication

### PR DESCRIPTION
**Pull request by ggw1991**
Added multiplication

### CODE REVIEW

File: strcalc.R [hunk 1]
```diff
@@ -6,6 +6,7 @@ computer<-function(input_string){
   result <- switch(operator,
                    '+' =num0+num1,
                    '-' =num0-num1,
+                   '*' =num0*num1,
```
Comment (ALBJ) [inline strcalc.R:9]: Wow!!!!

Comment (ALBJ) [inline strcalc.R:9]: Hey there, thank you! Could you do divide as well? :)

Comment (ggw1991) [inline strcalc.R:9]: You can do. Lazy

Comment (ALBJ) [inline strcalc.R:9]: Haaaah okay

**Pull request merged**